reana_job_controller/config.py

```diff
@@ -19,9 +21,20 @@
 """Root path of the shared volume ."""
 
 JOB_BACKENDS = {
-    'kubernetes': KubernetesJobManager
+    'kubernetes': KubernetesJobManager,
+    'htcondorcern': HTCondorJobManagerCERN
 }
 """Supported job backends and corresponding management class."""
 
 DEFAULT_JOB_BACKEND = 'kubernetes'
 """Default compute backend for job submission."""
+
+HTCONDOR_SUBMISSION_JOB_IMG = os.getenv('HTCONDOR_SUBMISSION_JOB_IMG',
```

Comment:
This is CERN-specific, so should be (1) either living in that class, or (2) renamed to `HTCONDORCERN_....`.
In this way we can always `grep -i htcondorcern` (the text equal to backend value) in order to see whether there are HTCondorCERN-specifics parts throughout the codebase...

Reply:
... ticketised for later as #157